caffe2/python/tutorials/py_gen/MNIST.py

```diff
@@ -10,9 +10,11 @@
 
 # # MNIST
 # 
-# In this tutorial, we will show you how to train an actual CNN model, albeit small. We will be using the good ol' MNIST dataset and the LeNet model, with a slight change that the sigmoid activations are replaced with ReLUs.
+# In this tutorial, we will show you how to train an actual convolutional neural network (a CNN) model, albeit small. We will be using the good old MNIST dataset. The problem here is hand written digits recognition. Each sample is a 28x28 picture of a hand written single digit and the label is a digit from 0 to 9. 
```

Comment:
In this tutorial, we will show you how to train a small convolutional neural network (a CNN) model.  We will be training the model on the MNIST dataset, which consists of labeled handwritten digits.  Each sample is a 28x28 picture of a handwritten single digit and the label is a digit from 0 to 9.